.eslintrc

```diff
@@ -3,6 +3,8 @@ reportUnusedDisableDirectives: true
 
 ignorePatterns:
   - /web_src/js/vendor
+  - /templates/base/head_script.tmpl
+  - /templates/repo/issue/view_content/pull.tmpl
```

Comment:
I would against to do these ignores.

Instead, please use `eslint-disable` accurately.

Reply:
Some `eslint-disable` should be able to be written clearly in templates, but the author doesn't respond yet.

https://github.com/eslint/eslint/discussions/15856

Reply:
I wasn't able to get `eslint-disable` to properly work in vue html files. Might have something todo about the plugin setup, I'm not sure.

https://github.com/vuejs/eslint-plugin-vue/issues/260

Reply:
So it seems the reason these existing `<!-- /* eslint-disable */ -->` comments work is because of the parser that's loaded via `plugin:vue/recommended`. As soon as I disables that that, the file fails to parse.

On the other hand, the vue parser can not deal with hashbang in the node scripts, only eslint's own parser can.